packages/translations/src/languages/uk.ts

```diff
@@ -215,9 +215,9 @@ export const uk: Language = {
       leaveWithoutSaving: 'Вийти без збереження',
       light: 'Світла',
       livePreview: 'Попередній перегляд',
-      loading: 'Загрузка',
+      loading: 'Завантаження',
       locale: 'Локаль',
-      locales: 'Переклади',
+      locales: 'Локалі',
```

Comment:
idk, is it right actually?
from chatgpt:
> "локалі" - це українське слово, яке вживається для позначення місцевих закладів, кафе, ресторанів або пабів, особливо часто вживається серед молоді.

i doesn't seem right here, i guess "Мови" could be.
And "Локаль" -> "Мова" as well.

Reply:
It depends, generally - yeah, I truly agree with you, but in the context of the content localization `локаль` seems to me more clear/descriptive.

Here are the few variations of the possible `locale` word translation:
`Переклад` - more about `translation` word, not `localization`
`Мова` - direct translation of the word `language`, and it's the most correct one (because generally used in Ukrainian), but in the case with Ukrainian language context it's not descriptive enough of the content localization
`Локаль` - it's really not as correct as `мова` (because it's an adopted word from English), but in the context of content localization it's the most descriptive one, and _feels the most natively \*imho\*_ in the context of the content localization

P. S.: ChatGPT lies a little bit, it's not that widely used among the youth, and generally not used to describe some physical place :)

Reply:
> It depends, generally - yeah, I truly agree with you, but in the context of the content localization `локаль` seems to me more clear/descriptive.
> 
> Here are the few variations of the possible `locale` word translation: `Переклад` - more about `translation` word, not `localization` `Мова` - direct translation of the word `language`, and it's the most correct one (because generally used in Ukrainian), but in the case with Ukrainian language context it's not descriptive enough of the content localization `Локаль` - it's really not as correct as `мова` (because it's an adopted word from English), but in the context of content localization it's the most descriptive one, and _feels the most natively *imho*_ in the context of the content localization
> 
> P. S.: ChatGPT lies a little bit, it's not that widely used among the youth, and generally not used to describe some physical place :)

I just haven't heard someone uses this word :). Anyway if someone doesn't like it there's an option to override it in the Payload config, so that's fine.